src/main/java/com/example/bookstore/service/cartitem/CartItemService.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bookstore.service.cartitem;
+
+import com.example.bookstore.dto.cartitem.CreateCartItemRequestDto;
+import com.example.bookstore.model.CartItem;
+import com.example.bookstore.model.ShoppingCart;
+
+public interface CartItemService {
+    CartItem save(CreateCartItemRequestDto createCartItemRequestDto,
+                  ShoppingCart shoppingCart);
+
+    CartItem findById(Long id);
```

Comment:
We will use `find` when the `Optional` should be returned
```suggestion
    CartItem getById(Long id);
```